heroic/hc_garothi_worldbreaker.simc

```diff
@@ -1,21 +1,25 @@
 # movement
 # decimation comes out about every 10s during the phases where the boss is active, will make a forced event every 30s (i.e. youll get targeted by every 3rd decimation)
 # first set before adds come out
-raid_events+=/movement,cooldown=30,duration=5,last=60
+raid_events+=/movement,cooldown=31,duration=5,last=85,player_chance=0.3,players_only=1
+# Surging Fel during first add phase
+raid_events+=/movement,cooldown=20,duration=2,first=85,last=110,player_chance=0.2,players_only=1
```

Comment:
Same thing here on the player_only=1 stuff, probably either remove the mechanic or remove players only 1